sklearn/tests/test_cross_validation.py

```diff
@@ -1005,26 +1014,6 @@ def test_shufflesplit_reproducible():
     assert_array_equal(list(a for a, b in ss), list(a for a, b in ss))
 
 
-def test_safe_split_with_precomputed_kernel():
```

Comment:
Just curious, why did you remove this test and the one below. Were they failing?

I thought you meant to keep all the old tests because of what you say in the PR description:

> NOTE: Lot more lines can be removed from more redundant tests, but I feel leaving them as such (which will cost a 3-4 extra seconds) will ensure that we haven't regressed on the old modules)